### DISCUSSION TITLE
"Download > HTML" supposed to lose embelishments?

### DISCUSSION THREAD

**Discussion by infinite-persistence** [outline/outline#8061] [General] [answered]
Unlike the static site version of a doc, the HTML downloaded from the context menu seems to lose items like:
- Notice icons
- Image captions

and ignoring the defined `toDOM`s?

<img src="https://github.com/user-attachments/assets/fccdd80e-4141-4bd0-a053-ab6ecb4ec1d5" width="400">

**Comment by tommoor** [comment, marked as answer]
Ideally it would be identical, the reason this is the case is that the HTML export option goes through the prosemirror HTML serializer (calls `toDOM` on each node) and does not include react-driven NodeViews. I believe it's fixable, just a little complicated – it would be nice to have a try soon tbh
